rnd/autogpt_builder/src/components/CustomNode.tsx

```diff
@@ -133,6 +147,137 @@ export function CustomNode({ data, id, width, height }: NodeProps<CustomNode>) {
     ));
   };
 
+  const generateInputHandles = (
+    schema: BlockIORootSchema,
+    nodeType: BlockUIType,
+  ) => {
+    if (!schema?.properties) return null;
+    let keys = Object.entries(schema.properties);
+    switch (nodeType) {
+      case BlockUIType.INPUT:
+        // For INPUT blocks, only show the 'value' property
```

Comment:
Is separate `case` for input and output necessary? They don't have any other properties anyway?

Reply:
Input blocks do not have connection handles.
Output blocks only have a connection handle for the `value` parameter

Reply:
i still don't understand what's the point. The default code should just draw the existing handles but I know you'd rather have simplicity so I hope that's justified!

Reply:
I want turn off that behaviour. So some inputs / outputs do not have handles drawn